prime-router/src/main/kotlin/tokens/OktaAuthentication.kt

```diff
@@ -18,9 +16,9 @@ const val oktaGroupPrefix = "DH"
 const val oktaSenderGroupPrefix = "DHSender_"
 const val oktaAdminGroupSuffix = "Admins"
 const val oktaSystemAdminGroup = "DHPrimeAdmins"
-const val oktaSubjectClaim = "sub"
 const val oktaMembershipClaim = "organization"
 const val envVariableForOktaBaseUrl = "OKTA_baseUrl"
+const val subjectClaim = "sub"
```

Comment:
Constants that are common between AuthenticatedClaims and OktaAuthentication could be pulled out into a AuthConstants.kt file

Reply:
So moved.